src/components/user/userModel.js

```diff
@@ -62,6 +62,15 @@ const UserModel = sequelize.define(
     role: {
       type: DataTypes.STRING,
     },
+    image_name: {
+      type: DataTypes.STRING,
+      default: "uservice-default-user-avatar.png",
+      allowNull: false,
+    },
```

Comment:
What's the point of `image_name`, It means name of user's avatar?

Reply:
When we upload an image from FE to BE, we just save the name of the image then the image source to S3. Whenever we need to get the image, we use getImageUrl(imageName, timeToLive) to get the image_url from S3 (will not make this link live forever, it's costly and since we don't have cache till now.

Reply:
Discussed in Slack